index.js

```diff
@@ -88,13 +88,13 @@ module.exports = function (opts) {
 
   async function parseBody(ctx) {
     if (enableJson && ((detectJSON && detectJSON(ctx)) || ctx.request.is(jsonTypes))) {
-      return await parse.json(ctx, jsonOpts);
```

Comment:
there has [some differences](https://jakearchibald.com/2017/await-vs-return-vs-return-await/) between `return await asyncFunction()` with `return asyncFunction()` especially with `try catch`, so I'd prefer always use `return await` instead of `return` to avoid potential risk of refactor.  Also `return await` make it more clearly that the function will return a promise.

Reply:
There is, but I'll argue its overhead rather than utility since there's no try-catch block within `parseBody`. I also do not see the benefit of overhead to document. But np, I'll close this.